mrq/job.py

```diff
@@ -350,6 +351,23 @@ def save_retry(self, retry_exc):
 
             self._save_status("retry", updates, exception=True)
 
+    def _save_traceback_history(self, status, trace, job_exc):
+        """ Create traceback history or add a new traceback to history. """
+        failure_date = datetime.datetime.utcnow()
+
+        new_history = {
+            "date": failure_date,
+            "status": status,
+            "exceptiontype": job_exc.__name__
```

Comment:
saving the worker ID would be great too. (get_current_worker(), which may return None)